pyaedt/desktop.py

```diff
@@ -298,12 +289,17 @@ class Desktop:
 
     >>> import pyaedt
     >>> desktop = pyaedt.Desktop("2021.1", NG=True)
-    >>> hfss = pyaedt.Hfss()
+    pyaedt Info: pyaedt v...
+    pyaedt Info: Python version ...
```

Comment:
This will eventually be removed (or appear as a logger output) as we should avoid any printing to stdout.  See #269.

Reply:
Definitely agree.